### DISCUSSION TITLE
Please add Arch Linux

### DISCUSSION THREAD

**Discussion by phantomhell** [cachewerk/relay#145] [Ideas]
Please add Arch Linux support for using Relay

**Comment by michael-grunder** [comment]
Seems reasonable to me. I'll look into it.

**Comment by tillkruss** [comment]
@phantomhell We need more information. What version are you using?

**Comment by phantomhell** [comment]
I am using:
Arch x86_64, 
Redis 7.2.5,
PHP 8.3

**Comment by tillkruss** [comment]
https://github.com/cachewerk/relay/pull/146
